src/reducer.ts

```diff
@@ -108,6 +108,17 @@ export function onNgrxForms<TState = any>(): { reducer: ActionReducer<TState>; t
   };
 }
 
+/**
+ * This function is an individualized version of onNgrxForms for use cases
+ * where either no root reducer exists or the state is nested below the top level of the store
+ */
+export function onNgrxForm(): { reducer: ActionReducer<any>; types: string[] } {
+  return {
+    reducer: (state, action) => isFormState(state) && { ...formStateReducer(state, action) },
```

Comment:
There's a couple of things wrong with this. If `isFormState` returns `false`, then the result of the call will be `false` instead of the original state. Also, the spread is not needed either and can in fact cause issues since you force an update of the state, even if the action does not change the form state.